database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -15,12 +15,16 @@ public function up()
     {
         Schema::create('users', function (Blueprint $table) {
             $table->id();
-            $table->string('name');
+            $table->string('fullname');
             $table->string('email')->unique();
+            $table->string('mobile')->unique();
             $table->timestamp('email_verified_at')->nullable();
-            $table->string('password');
+            $table->timestamp('mobile_verified_at')->nullable();
+            $table->string('password')->nullable();
+            $table->enum('status', ['active', 'inactive', 'pending', 'block'])->nullable();
```

Comment:
It should not to be nullable, please remove that

Reply:
line 24 nullable was removed and defult was added